docs/ref/contrib/admin/index.txt

```diff
@@ -1465,6 +1465,27 @@ templates used by the :class:`ModelAdmin` views:
 
     See also :ref:`saving-objects-in-the-formset`.
 
+.. warning::
+
+    All hooks that return a ``ModelAdmin`` property return the property itself
+    rather than a copy of their values. Dynamically modifying this value can
+    lead to surprising results.
+
+    Let's take :meth:`ModelAdmin.get_readonly_fields` as an example::
```

Comment:
```suggestion
    rather than a copy of its value. Dynamically modifying this value can lead
    to surprising results. Let's take :meth:`ModelAdmin.get_readonly_fields` as
    an example::
```
(sorry I made a grammar mistake)